lib/Query.js

```diff
@@ -135,7 +135,7 @@ Query.prototype.exec = function (next) {
       for (i = 0; i < rangeKey.values.length; i++) {
         val = rangeKey.values [i];
         keyConditions.AttributeValueList.push(
-          rangeAttr.toDynamo(val, true)
+          rangeAttr.toDynamo(val, true, Model, { updateTimestamps: false })
```

Comment:
Are we sure that ` { updateTimestamps: false }` is a valid option here? I'm assuming it is, but just want to verify.

Reply:
Yes, because this line is for converting the value of a range key in a query, `.where(rangeKey).gt(thisValue)`, to DynamoDB notation, and we really don't want to update it as we convert it.